sdk/keyvault/Microsoft.Azure.KeyVault/README.md

```diff
@@ -1,5 +1,19 @@
 # Microsoft Azure Keyvault SDK for .NET
 
+## New libraries available
+
+This library has been replaced by the following new Azure SDKs. You can read about the new Azure SDKs at https://aka.ms/azsdkvalueprop.
+
+The latest libraries to interact with the Azure KeyVault service are:
```

Comment:
"Key Vault" is the actual product name. We should use that more consistently in public documentation, at least.